packages/tools/devtools/devtools-view/src/test/SettingsView.test.tsx

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import { debug } from "jest-preview";
+import React from "react";
+
+import { SettingsView } from "../components/index.js";
+
+import { assertNoAccessibilityViolations } from "./utils/index.js";
+
+describe("SettingsView Accessibility Check", () => {
+	it("SettingsView is accessible", async () => {
+		const { container } = render(<SettingsView />);
+		await assertNoAccessibilityViolations(container);
+	});
+
+	it("Can tab/arrow navigate through the SettingsView", async () => {
+		render(<SettingsView />);
+
+		const user = userEvent.setup();
+		// Focus on the first interactive element (dropdown theme selector)
+		await user.tab();
+		const dropdown = screen.getByRole("combobox", { name: /Theme Selection Dropdown/ });
+		expect(dropdown).toHaveFocus();
+		// The dropdown theme options are divs but get compiled as buttons in the expect function and thus fail any expect calls.
+		await user.tab();
+		const privacyLink = screen.getByRole("link", { name: /Microsoft Privacy Statement/ });
+		expect(privacyLink).toHaveFocus();
+
+		const usageToggle = screen.getByRole("switch", { name: /Usage Telemetry Toggle/ });
+		await user.tab();
+		expect(usageToggle).toHaveFocus();
+		debug();
```

Comment:
Did you mean to leave in the `debug()`?

Reply:
It isn't strictly required but it is very helpful for debugging purposes because it allows you to see the HTML structure of the component when local testing fails and identify any issues with the rendered output.

I can remove it and the corresponding jest-preview library and instead add a note referencing this in the accessibility doc task